app/src/api/types/arrival.ts

```diff
@@ -1,7 +1,17 @@
 export type ArrivalState = 'predicted' | 'stopping' | 'planned' | 'departed';
 
+export type ArrivalType =
+  | 'bus-stop'
```

Comment:
@anbraten "An arrival type is a bus stop" doesn't really make sense. Could we improve the naming?

Reply:
Nice catch. I simply changed it to the existing vehicle-type now.